configs/components/facter.rb

```diff
@@ -197,11 +197,6 @@
 
   pkg.install_file ".gemspec", "#{settings[:gem_home]}/specifications/#{pkg.get_name}.gemspec"
 
-  if platform.is_windows?
-    pkg.directory File.join(settings[:sysconfdir], "facter", "facts.d")
-  else
-    pkg.link "#{settings[:bindir]}/facter", "#{settings[:link_bindir]}/facter"
-    pkg.directory File.join('/opt/puppetlabs', 'facter')
```

Comment:
On non-windows `settings[:install_root]` corresponds to `/opt/puppetlabs`. I figured it'd be easier to use the variable rather than hard-coding `/opt/puppetlabs`. That also lets us use that call on windows rather then having a similar-but-different facts.d directory defined